### ISSUE TITLE
[Dungeons]{Oculus][Boss]

### ISSUE THREAD

**Issue by Nenze** [gamefreedomgit/Frostmourne#293]
- Fight are too quick -> add more HP to bosses
- Bosses melee attacks seem to do low amounts of damage 
- Would be cool if bosses had more HP 
tldr make it harder xd

1,Drakos the Interrogator:
- Does not summon people before the explosion 

2.Varos Cloudstrider https://www.youtube.com/watch?v=va4Rn2p1T40
- Arcane Beam should move towards random player yet it focused shadowfiend so it will propably focus hunter pets etc.
- Energize Cores Varos  engerizes a portion of the platform tho it seems it does not work

3.Mage lord https://www.youtube.com/watch?v=Ee7rQk3fn8U
- Frostbomb damage seems okay 630 tick but it could be a bit higher so players actually will move with boss.
- Timebomb also seems to work ok
- Empowered Arcane Explosion -> Not sure if fight was too fast or he just simply didint use it

4.ley guardian https://www.youtube.com/watch?v=cP-PTcmjtOs
- Arcane barage and volley propably deal normal dungeon version damage not heroic one and even then damage should be a bit more so its a challange (cant check it with my details for more info cuz drake gets damaged not playter look at video for damage)
- Fight was 1 minute and he didint spawn whelps
- planar shift should be at 60% and 20% hp yet it was only at 60%
- anomaly ? not sure purple thing on video while planar shift is ongoing 2.5k damage but should do 42,750k

https://wowwiki-archive.fandom.com/wiki/Ley-Guardian_Eregos
https://wowpedia.fandom.com/wiki/Drakos_the_Interrogator
https://wowpedia.fandom.com/wiki/Ley-Guardian_Eregos
https://wowwiki-archive.fandom.com/wiki/Varos_Cloudstrider
https://wowpedia.fandom.com/wiki/Mage-Lord_Urom

**Comment by Nenze** [comment]
Also thrash deal low damage besides big golem dudes with mages

**Comment by UltraNix** [comment]
Fixed.
